src/app/services/backup-restore.service.ts

```diff
@@ -0,0 +1,48 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs/Rx';
```

Comment:
`import { Observable } from 'rxjs/Observable' instead of importing full library.